win32/src/job_object_helper.c

```diff
@@ -176,8 +176,113 @@ IMPLEMENT_MOCKABLE_FUNCTION(, int, job_object_helper_limit_cpu, THANDLE(JOB_OBJE
 
 IMPLEMENT_MOCKABLE_FUNCTION(, int, job_object_helper_set_job_limits_to_current_process, const char*, job_name, uint32_t, percent_cpu, uint32_t, percent_physical_memory)
 {
-    (void)job_name;
-    (void)percent_cpu;
-    (void)percent_physical_memory;
-    return -1;
+    /*Codes_S_RS_JOB_OBJECT_HELPER_19_013: [ If percent_cpu is greater than 100 then job_object_helper_set_job_limits_to_current_process shall fail and return a non - zero value.] */
+    if (percent_cpu > 100)
+    {
+        LogError("Invalid arguments: job_name=%s, percent_cpu=%" PRIu32 "", job_name, percent_cpu);
```

Comment:
>job_name [](http://example.com/codeflow?start=76&length=8)

we need to log with MU_P_OR_NULL.

Passing `NULL` as a string is C violation (and while Windows is fine with it, other platforms are not).